guide/airline_reservation/airline_reservation_service.bal

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerina/mysql;
+import ballerina/observe;
+import ballerina/runtime;
+import ballerina/sql;
+//import ballerinax/docker;
```

Comment:
Why is the commented code here? If it's not required, let's just remove it. Else please include a comment to describe the information about the commented code, and when we need to uncomment it.

Reply:
Required only when deploying the services in Docker. Necessary instructions are provided in the Readme file under **Deploying on Docker** section.